charts/images.yaml

```diff
@@ -17,7 +17,7 @@ images:
 - name: etcd-druid
   sourceRepository: github.com/gardener/etcd-druid
   repository: eu.gcr.io/gardener-project/gardener/etcd-druid
-  tag: "v0.1.9"
+  tag: "v0.1.10"
```

Comment:
Could you add the release notes from https://github.com/gardener/etcd-druid/releases/tag/v0.1.10 ?

Reply:
WDYM? It's already done, or? I've copied what was provided in #2115

Reply:
never mind, you have already added them. my bad